src/collections/_documentation/workflow/alerts-notifications/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Alerts & Notifications
+sidebar_order: 0
+---
+
+Sentry notifications are broadly categorized into alerts and non-alert notifications. Alerts can be sent to many supported [LINK: integrations]. Non-alert notifications only go to email.
+
+## Alerts
+
+Sentry provides both Metric Alerts and Issue Alerts. You define either type of alert per project.
+
+### Metric Alerts
+
+{% capture __alert_content -%}
+Metric Alerts are currently in beta. For more information about access to Metric Alerts, feel free to reach out at alerting-feedback@sentry.io.
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Metric alerts allow you to filter and set thresholds on all errors across a project. Use metric alerts for high-level monitoring of patterns or fine-grained monitoring of individual events. Metric alerts help express lightweight service-level objectives (SLOs) such as users affected by signup-page errors or volume of database errors.
```

Comment:
Ah, I see this one has some text after it about Issue alerts, maybe use the same text instead of mine.